bin/build.sh

```diff
@@ -1,73 +1,81 @@
 #!/bin/bash
 
-# Makes it so that all commands are displayed as they get executed
-set -x
```

Comment:
Why did you decide to drop this?

Reply:
I felt that it was unnecessary, seeing as the docker build pipeline just handles everything for you anyways. This was kinda used for debugging for you.